hardware/Comply-to-the-Regulations.md

```diff
@@ -0,0 +1,394 @@
+
+ ## 3.2 CE Regulation
+ Unlike FCC, CE is a “self-declaration” and not a certification. Since there is no certification, there is no modular approval / certification either. CE requires a declaration from the manufacturer that the products meet the requirements of the R&TTE or [RED](https://ec.europa.eu/docsroom/documents/23321/attachments/1/translations/en/renditions/native) directive. 
+ - A module manufacturer is responsible that the module is compliant and the end product manufacturer is responsible that the end product is compliant.Test reports of Silicon Labs modules are available and show compliance, but any end product needs its own evidence;
+ - The manufacturer of the end product (OEM) is responsible for the product being compliant.Silicon Labs strongly recommends to the OEM to perform radiated testing on the end product. The conducted test results can be inherited from Silicon Labs modules’ test reports. Typically, test evidence (reports) must be presented Optionally, NB (Notified Body) opinion can be asked as a formal “certificate”; 
+ - A product is tested according to harmonized standards: A harmonized standard is a European standard developed by a recognized European Standards Organization: CEN, CENELEC, or ETSI. It is created following a request from the European Commission to one of these organizations. Manufacturers, other economic operators, or conformity assessment bodies can use harmonized standards to demonstrate that products, services, or processes comply with relevant EU legislation.
+ There are RF, EMC, and Safety relevant standards for radio products. All harmonized standards with status [information](https://portal.etsi.org/webapp/WorkProgram/Frame_WorkItemList.asp?qSORT=DIRECTIVES&qHARMONIZED=True&butPredefined=Search&qDIRECTIVE=2014%2F53%2FEU&optDisplay=ALL&butHarmonized=Search);
+ - If harmonized standards are not followed, it is mandatory to use Notified Body;
+ - Manufacturer can use Notified Body also just to get more confidence about the compliance;
+ - Manufacturer maintains Technical Construction File (TCF)
+   - Test reports
+   - Description of the device (schematic, layout, BOM, photos, etc.)
+   - DoC
+   - NB opinion letter (if applied)
+ 
+# 4. Basic RF test items and limits for complying the regulations 
+Allthough there are RF, EMC and Safety categories need to be certified for regional regulations. It only cares about the RF category in this chapter. And Silicon Labs has SoCs,SiP,and PCB module products, the certification procedure or items are different among them. In this chapter, it only describes the test items and limits base on the end product with SoCs. For end products with SiP/PCB-Module, the finnal test items maybe reduced due to pre-certificated SiP/PCB-Module as described in previous section. As Silicon Labs' products contain sub-G frequency bands and 2.4GHz bands, it will introduce the test items and limits seperately in frequency bands due to the difference regulation requirements.
```

Comment:
Change "Allthough" to "Although" to remove extra l.  Change "categories need" to "categories which need".  Change "regulations.  It only" to "regulations, it only".  Add spaces to change "SoCs,SiP,and" to "SoCs, SiP, and".  Change "base on" to "based on".  Two instances to replace - with space, in changing "SiP/PCB-Module" to "SiP/PCB Module".  Change "finnal" to "final".  Change "pre-certificated" to "pre-certified".  Change "sub-G frequency bands and 2.4GHz bands" to "sub-GHz and 2.4GHz frequency bands".  Change "seperately" to "separately".

Reply:
Corrected.

Reply:
Corrected.